axiom/store.py

```diff
@@ -2074,23 +2074,9 @@ def getTypeID(self, tableClass):
         return self.transact(self._maybeCreateTable, tableClass, key)
 
 
-    def _maybeCreateTable(self, tableClass, key):
+    def _justCreateTable(self, tableClass):
         """
-        A type ID has been requested for an Item subclass whose table was not
-        present when this Store was opened.  Attempt to create the table, and
-        if that fails because another Store object (perhaps in another process)
-        has created the table, re-read the schema.  When that's done, return
-        the typeID.
-
-        This method is internal to the implementation of getTypeID.  It must be
-        run in a transaction.
-
-        @param tableClass: an Item subclass
-        @param key: a 2-tuple of the tableClass's typeName and schemaVersion
-
-        @return: a typeID for the table; a new one if no table exists, or the
-        existing one if the table was created by another Store object
-        referencing this database.
+        Just create the table for an Item subclass.
```

Comment:
Super nit-picky, I know, but most of the docstrings in Axiom are pretty good, so I'd like to see the usual epydoc (or sphinx or whatever is good these days) markup in here.

Reply:
Good catch; it's a private method, but given that we're calling it from another "friend" module, it definitely deserves better documentation.